queries.sql

```diff
@@ -1,3 +1,13 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+\d animals;
+SELECT * FROM animals;
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
```

Comment:
- [ ] It would be better if we fix this part of the query, the name `Pikachu` was not entered in the database in the `data.sql` file so obviously the query result will output wrong result. Kindly fix the screenshot as well